pom.xml

```diff
@@ -29,6 +29,7 @@
         <start-class>no.nav.familie.ef.iverksett.WebApplicationKt</start-class>
         <felles-kontrakter.version>2.0_20210416141710_97efcc5</felles-kontrakter.version>
         <token-validation-spring.version>1.3.5</token-validation-spring.version>
+        <kotlin-coroutines.version>1.3.9</kotlin-coroutines.version>
```

Comment:
Debug fungerer ikke hos meg med version 1.4.3, kan fjerne dette om det funker hos dere :)

Reply:
<img width="777" alt="Skjermbilde 2021-05-04 kl  10 33 12" src="https://user-images.githubusercontent.com/402915/116978554-250b5800-acc4-11eb-9105-900dca17960e.png">

Vet ikke om der er løsning eller hack, men dette fungerte for meg

Reply:
Takk! Dette funker :) Fikser pom :)